ingestion/src/metadata/ingestion/source/postgres.py

```diff
@@ -49,6 +60,37 @@ def create(cls, config_dict, metadata_config_dict, ctx):
         metadata_config = MetadataServerConfig.parse_obj(metadata_config_dict)
         return cls(config, metadata_config, ctx)
 
+    def get_databases(self) -> Iterable[Inspector]:
+        if self.config.database != None:
+            yield from super().get_databases()
+        else:
+            query = "select datname from pg_catalog.pg_database;"
+
+            results = self.connection.execute(query)
+
+            for res in results:
+
+                row = list(res)
+                try:
+
+                    logger.info(f"Ingesting from database: {row[0]}")
+                    self.config.database = row[0]
+                    self.engine = get_engine(self.config)
+                    self.connection = self.engine.connect()
+                    yield inspect(self.engine)
+
+                except Exception as err:
+                    logger.error(f"Failed to Connect: {row[0]} due to error {err}")
+
+    def get_table_fqn(self, service_name, schema, table_name) -> str:
+        return f"{service_name}.{self.config.database}_{schema}.{table_name}"
```

Comment:
Once the fqnSeparator changes are in - can you add FQNseparator instead of "." and remove the usage of underscore and add FQNseparator.